shared/js/l10n.js

```diff
@@ -2357,7 +2324,7 @@
     }
 
     if (typeof value === 'string') {
-      if (!entity.overlay) {
+      if (value.indexOf('<') === -1 && value.indexOf('&') === -1) {
```

Comment:
this will overlay strings like "Warsaw < San Francisco" and "Mail & News". I think it would make sense to test the next char not being space here.